docs/design/setMTU.md

```diff
@@ -0,0 +1,20 @@
+# Set custom MTU parameter
+
+## Goals
+  - Allow specifying MTU value for networks used by container jobs (useful for docker-in-docker scenarios in k8s cluster).
+ 
+## Configuration
+
+You need to set the environment variable AGENT_MTU_VALUE to set the MTU value, after that config and run the self-hosted agent.
```

Comment:
Re-configuration is not required for agent knobs being set via environment variables - only agent restart - could you please update?